endoreg_db/models/metadata/model_meta.py

```
@@ -128,6 +128,20 @@ def create_from_file(
+    @classmethod
+    def setup_default_from_huggingface(
+        cls: Type["ModelMeta"],
+        model_id: str,
+        task: str = "image-classification",
+        labelset_name: Optional[str] = None,
+    ) -> "ModelMeta":
+        """
+        Downloads a pretrained model from Hugging Face and initializes ModelMeta automatically.
+        """
+        from . import model_meta_logic as logic
+        return logic.setup_default_from_huggingface_logic(cls, model_id, task, labelset_name)
```

Pylint (4.0.1)

[error] Too many positional arguments for function call

(E1121)